packages/sdk-js/src/GrowthBook.ts

```diff
@@ -1464,6 +1551,57 @@ export class GrowthBook<
     return false;
   }
 
+  private _isExperimentBlockedByContext(
+    experiment: Experiment<AutoExperimentVariation>
+  ): boolean {
+    if (
+      experiment.changeType === "visual" &&
+      this._ctx.disableVisualExperiments
+    )
+      return true;
+
+    if (experiment.changeType === "redirect") {
```

Comment:
fyi: relies on new field `changeType` which may not exist on legacy cached payloads